other/remove-hostpath-volumes/pods.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod01
+  namespace: remove-hostpathvols-ns
+spec:
+  automountServiceAccountToken: false
+  containers:
+  - name: busybox
+    image: busybox:1.35
+    volumeMounts:
+    - mountPath: /test-pd
+      name: foo
+    - mountPath: /foo
+      name: empty
+  - name: busybox02
+    image: busybox:1.35
+    volumeMounts:
+    - mountPath: /foo
+      name: empty
+    - mountPath: /test-pd
+      name: bar
+  - name: busybox03
+    image: busybox:1.35
+    volumeMounts:
+    - mountPath: /foo
+      name: empty
+  volumes:
+  - name: foo
+    hostPath:
+      path: /data
+  - name: bar
+    hostPath:
+      path: /data/junk
+  - name: empty
+    emptyDir:
+      medium: memory
+      sizeLimit: 20Mi
```

Comment:
Swap the order here, please. We had a problem in the past where some logic wasn't working properly if these were separate. We want to ensure that doesn't happen again. Seems like a minor thing but it hurt us in the past.

Reply:
Sure! Changing.